packages/picker/stories/picker.stories.ts

```diff
@@ -156,3 +157,33 @@ export const initialValue = (): TemplateResult => {
         </sp-picker>
     `;
 };
+
+export const custom = (): TemplateResult => {
+    return html`
+        <sp-picker
+            style="width: 400px;"
+            @change="${(event: Event): void => {
+                const picker = event.target as Picker;
+                action(`Change: ${picker.value}`)();
+            }}"
+            label="Pick an state"
```

Comment:
```suggestion
            label="Pick a state"
```
Just a little nitpick :)